libs/blocks/caas-config/caas-config.js

```diff
@@ -639,7 +679,7 @@ const getInitialState = () => {
   // /* c8 ignore next 2 */
   if (!state) {
     const lsState = localStorage.getItem(LS_KEY);
-    if (lsState) {
+    if (lsState?.includes('filtersCustom')) {
```

Comment:
Do we really only want to parse the state if it has customfilters?

Reply:
Yes. This is needed only to support legacy users, with and older localStorage formatted state.

Reply:
Let's add a comment here explaining why we're doing this

Reply:
@chrischrischris 
The CAAS configurator saves the current state with the user's selected options in `localStorage `for persistency. 
Since the current implementation does not use the `filtersCustom `option, that entry is missing from the saved state and the preact code fails without this check. Originally we just checked if the localStorage state (lsstate) existed. not we check if it exists AND if it contains the new property filtersCustom before using it.